server/api/controllers/admin.js

```diff
@@ -3253,6 +3253,94 @@ const deleteAnnouncement = (req, res) => {
 			return res.status(err.statusCode || 400).json({ message: errorMessageConverter(err, req?.auth?.sub?.lang) });
 		});
 };
+
+
+const getExchangeUserRoles = (req, res) => {
+	loggerAdmin.verbose(req.uuid, 'controllers/admin/getExchangeUserRoles/auth', req.auth.sub);
+
+	toolsLib.user.getExchangeUserRoles()
```

Comment:
this can be retrieved from memory. Add this to init. When the roles are updated or new one added we can update that with websocket